argilla/src/argilla/responses.py

```diff
@@ -193,7 +193,10 @@ def _compute_status_from_answers(self, answers: List[Response]) -> ResponseStatu
             return ResponseStatusModel(next(iter(statuses)))
         return ResponseStatusModel.draft
 
-    def _compute_user_id_from_answers(self, answers: List[Response]) -> UUID:
+    def _compute_user_id_from_answers(self, answers: List[Response]) -> Optional[UUID]:
+        if len(answers) == 0:
+            return None
+
```

Comment:
Why do we call this `answers` and not `responses`? I prefer to limit the use of `answers` to situations where a thing could be either a response or suggestion.